mln/tests/services/test_misc.py

```diff
@@ -85,6 +100,24 @@ def test(self):
 		self.assertFalse(self.user.inventory.filter(item_id=self.REQUIREMENT_ID).exists())
 		self.assertTrue(self.user.inventory.filter(item_id=self.ITEM_ID, qty=1).exists())
 
+class CraftMasterpiece_Ok(TestCase):
+	SETUP = has_masterpiece_blueprint, has_masterpiece_requirement
+	
+	def test(self):
+		use_blueprint(self.user, self.MASTERPIECE_BLUEPRINT_ID)
+		self.assertFalse(self.user.inventory.filter(item_id=self.MASTERPIECE_REQUIREMENT_ID).exists())
+		self.assertTrue(self.user.inventory.filter(item_id=self.MASTERPIECE_ITEM_ID, qty=1).exists())
+		self.assertTrue(self.user.profile.rank == 1)

```

Comment:
Use `assertEqual` here so in the case of a mismatch the value is printed.

```suggestion
		self.assertEqual(self.user.profile.rank, 1)
```